genyrator/templates/sqlalchemy/model_to_dict.j2

```diff
@@ -99,3 +96,19 @@ def _recurse_on_model_or_list(
             return_immediately=return_immediately,
         )
 
+
+def deep_merge(dict1: Mapping[str, Any], dict2: Mapping[str, Any]):
```

Comment:
Given that this is a pretty generic function with a pretty specific use case I would be inclined to make it module local (ie. underscore prefix).